homeassistant/components/wiz_light/__init__.py

```diff
@@ -2,10 +2,13 @@
 import logging
 
 from pywizlight import wizlight
+from pywizlight.exceptions import WizLightConnectionError, WizLightTimeOutError
 
+from homeassistant.components.wiz_light.light import WizBulb
```

Comment:
```suggestion
from .light import WizBulb
```

Would be cleaner to put it in its own file ..maybe `device.py`

Reply:
Actually ignore this.  I didn't realize `WizBulb` was an entity.  We shouldn't import it here.  I'll add another suggestion in light.py to solve this in another way.